constraints.txt

```diff
@@ -0,0 +1 @@
+-c https://dist.plone.org/release/6.0.13/constraints.txt
```

Comment:
Your PR targets master which is only used for Plone 6.1

.meta.toml says 6.1-latest -- did you apply these changes using plone/meta?

Reply:
@davisagli what branch should be used instead?

Reply:
@jackahl I am not sure, it will be quite a mess if we have to update a version in each package after every Plone release. Personally I still use https://github.com/plone/buildout.coredev for working on core Plone backend packages.

Reply:
This should indeed be `-c https://dist.plone.org/release/6.1-dev/constraints.txt` to be in line with what `.meta.toml` and `tox.ini` say.

And indeed, I would use `buildout.coredev` for working on core Plone packages.  Note that since recently this also has support for installing with pip/mxdev, though it may need some fine tuning in practice.

Reply:
@mauritsvanrees This is still wrong.

Reply:
Sorry, I merged PR #701, but did not realise that the current PR was part of it and still had open issues.

Reply:
@mauritsvanrees ah yes, I see how that happened by accident